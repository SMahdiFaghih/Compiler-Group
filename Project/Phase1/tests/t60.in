if (00434143 + 0X1f30dax - b) {
    While = If;
}