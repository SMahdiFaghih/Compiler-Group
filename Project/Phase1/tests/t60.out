if
(
T_INTLITERAL 00434143
+
T_INTLITERAL 0X1f30da
T_ID x
-
T_ID b
)
{
T_ID While
=
T_ID If
;
}